conf/bookkeeper.conf

```diff
@@ -158,6 +158,10 @@ numReadWorkerThreads=8
 # (i.e. recovery reads and adds, and fencing).
 numHighPriorityWorkerThreads=8
 
+# This is the number of threads used by Netty to handle TCP connections.
+# Default is 2 *  Runtime.getRuntime().availableProcessors()
+# serverNumIOThreads=
```

Comment:
We should leave uncommented, so it will be easier to override in K8S with just env variables.